src/clojure/nrepl/cmdline.clj

```diff
@@ -153,12 +184,15 @@
         (System/exit 0))
       ;; otherwise we assume we have to start an nREPL server
       (let [bind (options "--bind")
+            auth-wrapper (some-> (options "--authorization-wrapper")
+                                 resolve-auth-wrapper-or-die)
             ;; if some handler was explicitly passed we'll use it, otherwise we'll build one
             ;; from whatever was passed via --middleware
             handler (and (options "--handler") (read-string (options "--handler")))
             middleware (and (options "--middleware") (read-string (options "--middleware")))
             handler (if handler (handler) (build-handler middleware))
-            server (start-server :port port :bind bind :handler handler)
+            server (apply start-server :port port :bind bind :handler handler
```

Comment:
I don't think `apply` is needed here, as you can safely pass a nil auth-wrapper and this will result in the default being used.